examples/AWSDriverExample/src/main/java/software/amazon/AwsIamAuthenticationMariadbExample.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *
+ *     Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License").
+ *     You may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package software.amazon;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import software.amazon.jdbc.PropertyDefinition;
+
+public class AwsIamAuthenticationMariadbExample {
+  public static final String MYSQL_CONNECTION_STRING =
+      "jdbc:aws-wrapper:mysql://db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com:3306?permitMysqlScheme";
+
+  private static final String USERNAME = "john_smith";
+
+  public static void main(String[] args) {
+
+    final Properties properties = new Properties();
+
+    // Enable AWS IAM database authentication and configure driver property values
+    properties.setProperty(PropertyDefinition.USER.name, USERNAME);
+    properties.setProperty(PropertyDefinition.PLUGINS.name, "iam");
+    properties.setProperty("sslMode", "verify-ca");
+    properties.setProperty("serverSslCert", "path/to/certificate.pem");
+
+    // Attempt a connection
+    try (Connection conn = DriverManager.getConnection(MYSQL_CONNECTION_STRING, properties);
+        Statement stmt1 = conn.createStatement();
+        ResultSet rs = stmt1.executeQuery("SELECT 1")) {
+
+      while (rs.next()) {
+        System.out.println(rs.getString(1));
+      }
+    } catch (SQLException e) {
```

Comment:
Can you take a look at the postgres example and use a similar way to execute queries and print out results

Reply:
Now that I look at it I think the PG sample code might not compile due to missing imports, can you please fix it too.

Reply:
PG sample code is compiling